py/desisurvey/utils.py

```diff
@@ -141,9 +141,9 @@ def equ2gal_J2000(ra_deg, dec_deg):
     dec = np.radians(dec_deg)
 
     x = np.empty(3, dtype='f8')
-    x[0] = np.cos(ra) * np.cos(dec)
-    x[1] = np.cos(ra) * np.sin(dec)
-    x[2] = np.sin(ra)
+    x[0] = np.cos(dec) * np.cos(ra)
+    x[1] = np.cos(dec) * np.sin(ra)
+    x[2] = np.sin(dec)
 
     M = np.array([ [-0.054876, -0.873437, -0.483835],
```

Comment:
I added the source of the values in a second commit; it doesn't show the computation (it cites Green's "Spherical Astronomy"), but at the minimum, there are three numbers that cannot be derived from first principles and so have to be hard-coded, e.g. the three Euler angles (alpha, beta, gamma).

I also changed the test to compare against astropy rather than NED (I think the latter may be using current epoch rather than 2000, so results in differences of order 10^-4 rather than 10^-5).